addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4335,6 +4335,10 @@ export class OdooEditor extends EventTarget {
         ) {
             this.setContenteditableLink(link);
         }
+        const oEditableBanner = closestElement(ev.target, '.o_editable_no_shadow');
+        if (oEditableBanner && !oEditableBanner.isContentEditable) {
+            oEditableBanner.setAttribute('contenteditable', 'true');
+        }
```

Comment:
IMO, naming it `banner` would be good. Also does our banner will be contenteditable= false all the time? if yes then i thisnk we dont need to check  for this `!oEditableBanner.isContentEditable`. and this part of code will simply look like:
`       const banner = closestElement(ev.target, '.o_editable_no_shadow');
        if (banner) {
            banner.setAttribute('contenteditable', 'true');
        }
`

Reply:
@shsa-odoo When an already existing record is opened or when we refresh the record, only during that time `contentEditable` will be `false` otherwise, it will always be `true`.